3d_regression/densenet_training_array.ipynb

```diff
@@ -0,0 +1,372 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #2.    import math
</pre><p> </p><p>Please move the import into "setup import" section.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Project-MONAI/tutorials/pull/1737/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='9'/>

Reply:
<p>I would suggest instead to get rid of the <code>math</code> import and use Numpy only for that, so line 60 would be <code>rmse = np.sqrt(mse)</code>. The import  for <code>Regressor</code> should move to the <code>Setup Import</code> section.  </p>

Reply:
This has been done.